server/webapp/WEB-INF/rails.new/spec/webpack/models/dashboard/pipeline_spec.js

```diff
@@ -58,6 +58,13 @@ describe("Dashboard", () => {
 
       expect(pipeline.isFirstStageInProgress()).toBe(false);
     });
+
+    it("should return instance labels", () => {
+      const pipeline       = new Pipeline(pipelineJson);
+      const instanceLabels = pipeline.getInstanceLabels();
```

Comment:
@yankurk -- can we use pipeline counter instead of pipeline label to uniquely identify the pipeline instance.
Pipeline labels may not be unique across pipeline instances.

Reply:
Yes I missed that point. Putting in a fix.